spring-cloud-dataflow-server-cloudfoundry/src/main/resources/dataflow-server.yml

```diff
@@ -2,25 +2,24 @@ server:
   port: 9393
 management:
   contextPath: /management
-
+  security:
+    enabled: false
 info:
   app:
     name: "@project.artifactId@"
     description: "@project.name@"
     version: "@project.version@"
-
 security:
   basic:
     enabled: false
-#  oauth2:
-#    client:
-#      client-id: myclient
-#      client-secret: mysecret
-#      access-token-uri: http://127.0.0.1:9999/oauth/token
-#      user-authorization-uri: http://127.0.0.1:9999/oauth/authorize
-#    resource:
-#      user-info-uri: http://127.0.0.1:9999/me
-
+  oauth2:
```

Comment:
Yes - I left those commented out values there as a hint/additional help in case somebody wants to provide those values directly in a yaml file.